src/grids.jl

```diff
@@ -18,19 +18,19 @@ struct RegularCartesianGrid{T<:AbstractFloat, R<:AbstractRange} <: AbstractGrid{
     Ty::Int
     Tz::Int
     # Domain size [m].
-    Lx::T
-    Ly::T
-    Lz::T
+    Lx::FT
+    Ly::FT
+    Lz::FT
     # Grid spacing [m].
-    Δx::T
-    Δy::T
-    Δz::T
+    Δx::FT
+    Δy::FT
+    Δz::FT
     # Cell face areas [m²].
-    Ax::T
-    Ay::T
-    Az::T
+    Ax::FT
```

Comment:
We don't need areas and volumes right?

Reply:
That's true, we currently don't use them.

But thinking about it a bit more, I think we might want to start using them in PR #283. With the more general finite volume operators, we have areas that are computed like
```julia
@inline Az(i, j, k, grid) = Δx(i, j, k, grid) * Δy(i, j, k, grid)
```
although for `RegularCartesianGrid` and `VerticallyStretchedCartesianGrid` it might make more sense to use
```julia
@inline Az(i, j, k, grid) = grid.Az
```
to avoid an extra multiplication at the very lowest level.

Reply:
Not sure you'd see that benefit since I think our costs are dominated by accessing memory. But could be worth benchmarking. If it has no cost I think doing the multiplication is fine; makes the structs simpler which is well worth it. "Store information in functions".

Reply:
That's a good point. I'll remove them here as they're currently not being used. And I'll benchmark the two options when I work on merging in #283.

Reply:
It might make more sense for vertically stretched grid where `grid.ΔzF[k]` involves an extra memory access but we'll find out when we benchmark the vertically stretched grid.